src/main/java/com/ssomar/score/features/custom/cooldowns/CooldownsManager.java

```diff
@@ -27,16 +38,71 @@ public static CooldownsManager getInstance() {
     public Optional<String> onRequestPlaceholder(OfflinePlayer player, String params) {
         if (params.startsWith("cooldown_")) {
             UUID uuid = player.getUniqueId();
+
+            // Player uuid is found in current available cooldowns.
             if(cooldownsUUID.containsKey(uuid)){
                 boolean castInt = false;
+                boolean castMax = false;
+                String cooldownId = "";
+
+                // Find item slot to include below
+                if (params.contains("cooldown_slot")) {
+                    // if (!SCore.hasNBTAPI) {
+                    //     return Optional.of("You need NBT API!");
+                    // }
+                    // Extract the slot number using regex
+                    String slotNumber = params.replaceFirst(".*cooldown_slot(\\d+).*", "$1");
+                    int slot = Integer.parseInt(slotNumber);
+
+                    // Retrieve equipment for the slot
+                    if (player.isOnline() && player.getPlayer() != null) {
+                        ItemStack item = player.getPlayer().getInventory().getItem(slot);                   
+                        if (item != null) {
+                            if (item.hasItemMeta() && player != null) {
+                                ItemMeta iM = item.getItemMeta();
+            
+                                NamespacedKey key = new NamespacedKey(ExecutableItems.getPluginSt(), "ei-id");
+                                String eiid = iM.getPersistentDataContainer().get(key, PersistentDataType.STRING);
+            
+                                if (eiid != null) {
+                                    cooldownId += "EI:" + eiid + ":";
+                                    // get rid of the slot
+                                    params = params.replaceFirst("slot\\d+:", "");
+                                } else {
+                                    return Optional.of("Not an Executable Item: " + item.getItemMeta().getDisplayName());
+                                }
+                            } else {
+                                return Optional.of("No ItemMeta found or player is null");
+                            }
```

Comment:
Replace this part with 
if (SCore.hasExecutableItems) {
                    ExecutableItemObject item = new ExecutableItemObject(item);
                    if(item.isValid()){
                        // OK
                    }
                    else {
                        // KO
                    }
}